web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -63,8 +69,11 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
 The file is a markdown file, you can use `web/pdeps/0001.md` as a reference
 for the expected format.
 
-The initial status of a PDEP will be `Status: Under discussion`. This will be changed
-to `Status: Accepted` when the PDEP is ready and have the approval of the core team.
+The initial status of a PDEP will be `Status: Draft`. Once it is ready for discussion, the author(s)
+change it to `Status: Under discussion`, and the following are notified: core and triage teams
```

Comment:
Could clarify to ask your sponsor to ping the pandas-core/triage teams, since normally people can't ping/see the teams.

Reply:
We discussed creating some automation for the email and tag based on the label change. But I'll clarify for now.